aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -220,6 +223,28 @@ protected boolean isDBClusterStabilized(
         }
     }
 
+    protected boolean isDBClusterStabilizedAfterRemoveFromGlobalCluster(
+            final ProxyClient<RdsClient> proxyClient,
+            final ResourceModel model,
+            final String previousGlobalClusterIdentifier
+    ) {
+        final DBCluster cluster = fetchDBCluster(proxyClient, model);
```

Comment:
Please take this in a try-catch

Reply:
I can move the invocations withing the try-catch block, but I believe that our best option here is still to bubble exception up.

Reply:
I actually believe we don't want to catch the exceptions in `isDBClusterStabilized`. E.g. due to that code Throttling exception in that case would trigger failure to stabilise